<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>병리 이미지 분석 시스템</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="leftPanel" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="imageViewer">
          <property name="minimumSize">
           <size>
            <width>800</width>
            <height>600</height>
           </size>
          </property>
          <property name="text">
           <string>이미지 뷰어 영역</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rightPanel" native="true">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>AI 분석</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QGroupBox" name="heAIBox">
             <property name="title">
              <string>HnE AI</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QPushButton" name="btnSegmentation">
                <property name="text">
                 <string>조직 분할</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnClassification">
                <property name="text">
                 <string>암 분류</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnDetection">
                <property name="text">
                 <string>병변 검출</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="ihcAIBox">
             <property name="title">
              <string>IHC AI</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_3">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="progressLabel">
          <property name="text">
           <string>AI Progress</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>분석 결과</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTextEdit" name="resultText">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>파일</string>
    </property>
    <addaction name="actionOpenImage"/>
    <addaction name="actionSaveResults"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>보기</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionFitWindow"/>
    <addaction name="actionSlideInfo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenImage"/>
   <addaction name="actionSaveResults"/>
   <addaction name="separator"/>
   <addaction name="actionSlideInfo"/>
   <addaction name="actionFitWindow"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpenImage">
   <property name="icon">
    <iconset>
     <normaloff>../icon/Open.png</normaloff>../icon/Open.png</iconset>
   </property>
   <property name="text">
    <string>이미지 열기</string>
   </property>
  </action>
  <action name="actionSaveResults">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icon/save.png</normaloff>../icon/save.png</iconset>
   </property>
   <property name="text">
    <string>결과 저장</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset theme="열기">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>확대</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>축소</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionFitWindow">
   <property name="icon">
    <iconset>
     <normaloff>../icon/ScreenFill.png</normaloff>../icon/ScreenFill.png</iconset>
   </property>
   <property name="text">
    <string>화면 맞춤</string>
   </property>
  </action>
  <action name="actionSlideInfo">
   <property name="icon">
    <iconset>
     <normaloff>../icon/Info.png</normaloff>../icon/Info.png</iconset>
   </property>
   <property name="text">
    <string>슬라이드 정보</string>
   </property>
   <property name="toolTip">
    <string>슬라이드 정보 보기 (MPP, 배율, 크기 등)</string>
   </property>
  </action>
  <action name="actionSlideInfo_2">
   <property name="text">
    <string>actionSlideInfo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
